.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-
```

Comment:
I just noticed this render to Italic. Can you make it bold (use 2 stars ** https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)